dockerfiles/Dockerfile_full

```diff
@@ -10,7 +9,9 @@ RUN apt-get install --assume-yes \
   ./nodejs.deb \ 
   openjdk-11-jdk-headless=11.0.4+11-1ubuntu2~18.04.3 \
   jq \
-  && npm install --global sfdx-cli@${SALESFORCE_CLI_VERSION} \
+  && mkdir ~/.sfdx-cli \
+  && wget -qO- https://developer.salesforce.com/media/salesforce-cli/sfdx-linux-amd64.tar.xz | tar xJ -C ~/.sfdx-cli --strip-components 1 \
```

Comment:
This script will tar CLI source files to `~/.sfdx-cli` and the install script will add `sfdx` to `usr/local/bin` to make it globally available. There is no tar file to clean up at the end of this script.

Reply:
But this doesn't have the version info in it unlike the previous method of  `npm install --global sfdx-cli@${SALESFORCE_CLI_VERSION}`

How do we guarantee that what we are installing matches the version specified?

Reply:
good catch. this works because we always get the latest version and currently we build the image whenever a new version is available. This would also mean that the version we pass in and the version of CLI that is installed could possibly be different. we need to take care of that.

Reply:
Yeah I guess it would be good to have some check on that, I seem to recall reassurance from the CLI team that the manifest will always be updated when they publish the next version.

So for the stated image version and the published version in the manifest to be off, @jgrantham-sf (for example) would have to release the latest version but then have something go wrong with the manifest updating?
I seem to remember there were some issues with the manifest updating in the past but I don't know how often that happens or if we are unaware of when that happens.